scripts/console.ts

```diff
@@ -410,22 +426,80 @@ function etherscanTx(net: string, txHash: string): string {
 }
 
 function askForUsage(): string {
-    const usage = [CLONE_USAGE, DEV_USAGE];
-    const network = keyInSelect(usage, `Please enter your intended usage`, { cancel: true });
-    return usage[network];
+    const usageOpts = [CONTRIBUTOR_USAGE, MAINTAINER_USAGE, DEV_USAGE];
+    const usageChoice = keyInSelect(usageOpts, `Please enter your intended usage`, {
+        cancel: true,
+    });
+
+    switch (usageOpts[usageChoice]) {
+        case CONTRIBUTOR_USAGE:
+            console.log(
+                `\n\nYou will be asked to enter the address of the already deployed GitConsensus, Token, or Governor ` +
+                    `contracts that you want to use on ${network.name}. These will be defaulted from the list of the official deployed ` +
+                    `contracts on ${network.name} which can be found in this repository's 'deployments.json' file.`,
```

Comment:
Nit: Add a comma before the "which" to separate the independent clause from the dependent clause.

Reply:
fixed